### PR TITLE
fix: Impossible configure Alma if PS Account version is lower  5.0

### PR DESCRIPTION
**Pull request by Benjamin-Freoua-Alma**
### Reason for change

[Linear task](https://linear.app/almapay/issue/ECOM-2162/zepelucheclubcom-500-internal-sever-error)

### Code changes

We added a verification version of PS Account. If PS Account was lower that 5.0 PS Account Service returned an error that we didn't handle.
Now if the version is lower 5.0 we suggest to install the PS Account module (default message)

### How to test

_As a reviewer, you are encouraged to test the PR locally._

Install Ps Account lower that 5.0.
Install our module.
Try to configure it

### Checklist for authors and reviewers

- [ ] The title of the PR uses business wording, not technical jargon, for the changelog readers to understand it
- [ ] The PR implements the changes asked in the referenced task / issue
- [ ] The automated tests are compliant with the [testing strategy](https://www.notion.so/almapay/Backend-testing-strategy-06c642cec1bf47b9b8feca3a91ea8d4a)
- [ ] You understand the impact of this PR on existing code/features
- [ ] Documentation is updated (API, developer documentation, ADR, Notion...)

### Non applicable
- [ ] The tests are relevant, and cover the corner/error cases, not only the happy path
- [ ] The changes include adequate logging and Datadog traces

### CODE REVIEW

File: alma/alma.php [hunk 1]
```diff
@@ -604,10 +624,12 @@ public function getContent()
      * @return bool
      *
      * @throws \Alma\PrestaShop\Exceptions\CompatibilityPsAccountException
+     * @throws \PrestaShop\PsAccountsInstaller\Installer\Exception\ModuleNotInstalledException
      */
     public function renderPSAccount()
     {
         $this->checkCompatibilityPSModule();
+        $this->checkPsAccountCompatibility();
```
Comment (olance) [inline alma/alma.php:632]: The method names are a bit confusing, maybe you should:
- Only call `$this->checkPsAccountCompatibility();` here
- Rename `checkCompatibilityPSModule` to `checkPSAccountsPresence` or something similar
- Call it from `checkPsAccountCompatibility` as a first check

Comment (Benjamin-Freoua-Alma) [reply alma/alma.php]: @olance I agree with this. I updated it

**Review by olance — CHANGES_REQUESTED** [review]
Can you do a search & replace of `PsAccount` to `PsAccounts`? 😅
Just so that we're consistent with the real name.

**Comment by Benjamin-Freoua-Alma** [comment]
> Can you do a search & replace of `PsAccount` to `PsAccounts`? 😅 Just so that we're consistent with the real name.

@olance I replaced it 👍

**Review by olance — APPROVED** [review]

**Pull request merged**